### DISCUSSION TITLE
check_response on response text

### DISCUSSION THREAD

**Discussion by AntonioSun** [smicyk/groovy-jmeter#80] [Q&A] [answered]
I want to check each response for keyword "Wrong" and mark the request fail if found. Like this:

<img width="560" alt="image" src="https://user-images.githubusercontent.com/2840074/172633713-b9a31b0d-0220-4fc8-9b7e-4ec11f6beaed.png">

How to do it in [groovy-jmeter](https://github.com/smicyk/groovy-jmeter)?

**Comment by smicyk** [comment, marked as answer]
I think it should be something like:

```groovy
check_status applyTo: 'parent', {
    text(not) contains 'Wrong'
}
```

**Reply by AntonioSun** [reply]
Hmm...

I've tried 

```
      check_response applyTo: 'parent', {
        text(not) contains 'Wrong'
      }      
```

instead and it is working.

**Comment by smicyk** [comment]
Yes, my mistake, the keyword name is check_response

**Reply by AntonioSun** [reply]
Oh, and a related follow up question. What's the syntax to check for more than one cases? E.g., for `check_response` to get:

<img width="548" alt="image" src="https://user-images.githubusercontent.com/2840074/176915443-48363912-19d7-4243-8f16-b4e742045f61.png">

I intuitively tried the following but it is not working:

```
      check_response {
        status() eq 200 or
        status() eq 302
      }
```

**Reply by smicyk** [reply]
Should be more like:

```
check_response {
  status() eq 200 or 302
}
```

**Reply by AntonioSun** [reply]
Bingo! works great. thx!
